spikes/data/dataservice/tests/README.md

```diff
@@ -0,0 +1,21 @@
+# Test Docs
+
+- TODO - Add Details about how this works in ci-cd
+
+- You can run a complete end-to-end test using webv
+  - You must create your secrets first
+  - You will need a second bash console.
+
+```bash
+
+# from src/tests (first console)
+./runtests
+
+# wait for Application started. Press Ctrl+c to shutdown.
+
+# from TestFiles (second console)
+webv -s localhost:4120 -f baseline.json
+
```

Comment:
Should this use dotnet run with loderunner instead?  I think should when we move out of spikes, but for spikes folder probably easier to use webv.  Thoughts?

Reply:
There's a task on the board to change.